bes_theme/assets/data/OSSP-legend.js

```diff
@@ -0,0 +1,567 @@
+[
+  {
+    "TOSS-PM": {
+
+        "bes_oss_tech_stack": {
+          "DO" : {
+                  "id": 3333824398,
+                  "node_id": "MDU6TGFiZWwzMzMzODI0Mzk4",
```

Comment:
@jobyko @asa1997  only "id" & possibility "name" or "url"  is required as other details are present in labels json ,

Reply:
Will keep, id and name alone for the headings in the upcoming PR